homeassistant/__main__.py

```diff
@@ -356,7 +356,8 @@ def try_to_restart() -> None:
 
 def main() -> int:
     """Start Home Assistant."""
-    monkey_patch_asyncio()
+    if not sys.version_info >= (3, 5):
```

Comment:
Also, since `version_info` also contains `releaselevel` and `serial`, you should reference `sys.version_info[:3]`.